src/components/info-list.jsx

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 import '../styles/components/info-list.scss';
 
-const InfoList = ({ infoData }) => (
-  <div className="info-list">
+const InfoList = ({ infoData, columns = false }) => (
```

Comment:
The default value for `columns` is specified in the function parameters and also in `defaultProps`. Would be good to stick to one place.